### PR TITLE
Shuffles bedsheet attack actions a wee bit

### PR DESCRIPTION
**Pull request by flowercuco**
## About The Pull Request
my friends been doing makeshift surgery and gets mad about having to do harm intent to start surgery and then having to turn it off to do every single other surgery step and i agree, thats silly

~~instead, now, the bedsheet attack does not check for harm and ALWAYS COVERS with left mouse button and ALWAYS PREPARES SURGERY with right mouse button~~

to be more consistent all surgery initiators now work on left mouse click and covering with your bedsheet is the right mouse button

image

## Why It's Good For The Game

while obviously a lot of the time makeshift surgery IS about harming someone else its inconsistent with how all other surgery tools function and also isn't very clear at all. the screentip in addition to having them be separate buttons means that bedsheets two functions are overall much clearer and its surgery drape function will no longer lead to an assistant accidentally knifing someone when theyre trying to save someones life

## Changelog

:cl:
qol: added makeshift surgery screentip
qol: you no longer have to go in and out of harm intent to do surgery with a bedsheet
/:cl:

### CODE REVIEW

**Comment by dalmationer** [comment]
yippee!

**Review by MrMelbert — CHANGES_REQUESTED** [review]

**Comment by JohnFulpWillard** [comment]
I don't think I want surgery to be on right click, it would make it inconsistent with the surgical drapes.

**Comment by flowercuco** [comment]
> I don't think I want surgery to be on right click, it would make it inconsistent with the surgical drapes.

I agreed with you and melbert and current implementation is tucking ppl in is on right mouse click and all three surgery initiators prepare surgery on left mouse click

**Review by MrMelbert — CHANGES_REQUESTED** [review]

File: code/datums/components/surgery_initiator.dm [hunk 1]
```diff
@@ -19,9 +19,10 @@
 
 /datum/component/surgery_initiator/RegisterWithParent()
 	RegisterSignal(parent, COMSIG_ITEM_ATTACK, PROC_REF(initiate_surgery_moment))
+	RegisterSignal(parent, COMSIG_ITEM_REQUESTING_CONTEXT_FOR_TARGET, PROC_REF(add_item_context))
```
[the remark this answers is not in the document]
Comment (flowercuco) [reply code/datums/components/surgery_initiator.dm:25]: ok! i think i set it correctly?

**Pull request merged**